### DISCUSSION TITLE
Adding from iOS?

### DISCUSSION THREAD

**Discussion by mrbenns** [ArchiveBox/ArchiveBox#1452] [Q&A]
What options are there for adding a url from an iOS browser or share sheet?

**Comment by pirate** [comment]
I recommend using Pocket or Pinboard. It's what I do personally. 

You can then enable access to your feed in Pocket via RSS, then ArchiveBox can pull in URLs via that RSS feed every 24hr. 

Unfortunately I'm a total n00b when it comes to iOS/Swift development, but I'd welcome any contributions in that area if someone wants to build a small mobile connector app that uses our REST API!

I'm sure it's also possible to create an iOS Shortcut that sends a URL to the ArchiveBox REST API, but I haven't tried yet myself.

**Reply by IvanVas** [reply]
RSS is discontinued in Pocket, unfortunately.

**Reply by huyz** [reply]
@pirate  Is the REST API released or on a dev branch? What URL can we use to post?

**Reply by pirate** [reply]
POST to `/add` you can visit the page in the UI and submit + watch the network panel to see the request format. 

Full REST API is on dev but /add is available now on 0.7.2.

**Reply by Finkregh** [reply]
I didn't manage to find the docs how /add works with POST, what it expects. Any hints?

**Reply by pirate** [reply]
```bash
curl 'https://demo.archivebox.io/add/' \
  -H 'content-type: application/x-www-form-urlencoded' \
  --data-raw 'url=https%3A%2F%2Fexample.com&parser=auto&tag=&depth=0'
```

**Reply by Finkregh** [reply]
https://www.icloud.com/shortcuts/0d3a526e7d524447aa8c1bd63ac49014
You can use this shortcut ✨

**Reply by Finkregh** [reply]
@pirate should I create a PR for the docs with this?

**Comment by yhete** [comment]
Would it be possible to email a link to an ArchiveBox instance? I can't think of many ArchiveBox relevant content on ios that wouldn't allow me to share via email.

**Reply by pirate** [reply]
Sure you can probably vibecode something to do this, just check an IMAP inbox once per minute and add any URLs found to archivebox using the CLI or rest api.
